win_build.ps1

```diff
@@ -112,7 +112,7 @@ if (-Not $skipExporterCompile)
     }
 
     # remove unused collectors 
-    Remove-Item .\collector\* -Exclude $collectors
+    # Remove-Item .\collector\* -Exclude $collectors
```

Comment:
I would remove these two lines since we are no longer removing unused collectors

Also $collectors is not used, right?